.github/workflows/gradle.yml

```diff
@@ -0,0 +1,41 @@
+name: Build Gradle Tasks
```

Comment:
Thanks for contributing that, but can you open another PR to add the Actions config and the badge?
It would be easier to process and discuss as a separate change :)